tests/test_navbar.py

```diff
@@ -89,46 +102,32 @@ def test_logout_link(self, driver, page):
         login(driver)
```

Comment:
@DougCorell, This is a stray login that has been causing intermittent failures for a while. Let's go ahead and remove this.